src/libs/events.ts

```diff
@@ -20,6 +20,7 @@ const eventsList = {
   analysisEnableBeta: 'analysis:enable',
   analysisDisableBeta: 'analysis:disable',
   analysisCreate: 'analysis:create',
+  analysisLaunch: 'analysis:launch',
```

Comment:
I also don't understand why these changes are in this PR.

Reply:
I was looking at events.js, decided to upgrade it to Typescript, and fixed the TS errors that cropped up. This value was referenced, but never actually defined in `events`.

Reply:
I don't know if we want to do conversions where the file extension is just changed without types added.

I also don't understand how `analysisLaunch` is being referenced if it isn't defined. Peculiar.

Reply:
Because `events` wasn't typescript, there was no error when `https://github.com/DataBiosphere/terra-ui/blob/dev/src/analysis/AnalysisLauncher.js#L495` referenced it.

Reply:
Well actually, the place that calls the code isn't TypeScript so the fact that this file wasn't TypeScript doesn't fix the problem I think. But yes, the event should actually be defined (and/or IA should be given a heads-up)

Reply:
Gave IA a heads-up in slack!

Reply:
Opened #4276 to add types to libs/events.